### ISSUE TITLE
Failed to activate the resonance-ui theme

### ISSUE THREAD

**Issue by catnipped** [dktn/resonance-ui#3]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.9.9
**System**: Mac OS X 10.11.6
**Thrown From**: [resonance-ui](https://github.com/dktn/resonance-ui) package, v0.5.0
### Stack Trace

Failed to activate the resonance-ui theme

```
At '../../file-icons/styles/colors.less' wasn't found in /Users/ossianboren/.atom/packages/resonance-ui/styles/file-icons.less:-1:0

LessError: '../../file-icons/styles/colors.less' wasn't found
  at /Users/ossianboren/.atom/packages/resonance-ui/styles/file-icons.less:-1:0
```
### Commands

```
     -9:14.4.0 core:close (atom-text-editor.editor.is-focused)
     -9:03.6.0 application:open (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
  2x -8:36.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  5x -7:23.3.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -7:22 core:confirm (atom-text-editor.editor.mini.is-focused)
     -4:15.1.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -4:13.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -2:41.4.0 core:backspace (atom-text-editor.editor.mini.is-focused)
  2x -2:40.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {
    "closeEmptyWindows": false,
    "disabledPackages": [
      "highlight-column",
      "highlight-line",
      "spell-check",
      "language-gfm",
      "colorful-cursor",
      "fold-functions",
      "minimap"
    ],
    "themes": [
      "resonance-ui",
      "gruvbox-plus-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
an-old-hope-syntax, v0.5.0 (inactive)
atom-clock, v0.0.8 (active)
atom-material-ui, v1.3.5 (inactive)
atom-panda-syntax, v0.12.0 (inactive)
batman-syntax, v1.0.1 (inactive)
bensk-syntax, v2.7.1 (inactive)
blessed-dark-syntax, v0.3.0 (inactive)
cake-syntax, v0.5.3 (inactive)
clutterskull-syntax, v2.1.0 (inactive)
colorful-cursor, v0.3.1 (inactive)
contrast-syntax, v0.6.0 (inactive)
duotone-dark-red-syntax, v0.3.2 (inactive)
eclectic-syntax, v0.2.1 (inactive)
ekini-like, v0.9.1 (inactive)
firewatch-syntax, v1.0.1 (inactive)
fold-functions, v0.4.4 (inactive)
genesis-ui, v0.5.0 (inactive)
git-history, v3.2.0 (inactive)
git-plus, v5.16.2 (inactive)
git-time-machine, v1.5.3 (inactive)
gruvbox-plus-syntax, v1.2.5 (inactive)
gruvbox-syntax, v2.4.1 (inactive)
highlight-column, v0.5.1 (inactive)
highlight-line, v0.11.1 (inactive)
highlight-selected, v0.11.2 (active)
humane-syntax, v0.3.0 (inactive)
language-haskell, v1.9.7 (active)
language-lua, v0.9.11 (active)
language-markdown, v0.16.2 (active)
language-pico8, v0.1.0 (active)
local-history, v3.4.1 (active)
markdown-scroll-sync, v2.1.2 (active)
markdown-writer, v2.5.1 (inactive)
minimap, v4.24.7 (inactive)
minimap-pigments, v0.2.1 (active)
nimble-light-syntax, v0.2.0 (inactive)
nucleus-dark-ui, v0.9.3 (inactive)
pico8-syntax, v0.3.2 (inactive)
pigments, v0.33.1 (active)
pumpkin-syntax, v0.1.0 (inactive)
resonance-syntax, v0.1.0 (inactive)
resonance-ui, v0.5.0 (inactive)
rusty-dark-syntax, v0.6.1 (inactive)
scallahan-coffee-syntax, v1.2.0 (inactive)
spectrum-light-syntax, v0.0.5 (inactive)
symbols-tree-view, v0.13.2 (active)
tango-syntax, v0.4.1 (inactive)
union-syntax, v0.1.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.2 (inactive)
one-light-ui, v1.3.2 (inactive)
one-dark-syntax, v1.2.0 (inactive)
one-light-syntax, v1.2.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.0 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.0 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
find-and-replace, v0.198.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.1 (active)
metrics, v0.53.1 (active)
notifications, v0.64.1 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.238.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (inactive)
status-bar, v1.4.1 (active)
styleguide, v0.47.1 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.98.1 (active)
timecop, v0.33.1 (active)
tree-view, v0.208.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.32.2 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.36.2 (active)
language-gfm, v0.86.0 (inactive)
language-git, v0.13.0 (active)
language-go, v0.42.0 (active)
language-html, v0.44.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.22.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.0 (active)
language-less, v0.29.3 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.0 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.68.5 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.52.0 (active)
language-shellscript, v0.22.3 (active)
language-source, v0.9.0 (active)
language-sql, v0.21.1 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.8 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by dktn** [comment]
Ok, it's my fault. Basically the theme now assumes that this package is installed:
https://atom.io/packages/file-icons
I will try to fix it and let you know.

In meanwhile the simple workaround is to install file-icons package. If you don't like it you can disable it, but the paths will be found.

**Comment by dktn** [comment]
I fixed it in 0.7.0.
